stable/vulcan/templates/_dogstatsd.tpl

```diff
@@ -0,0 +1,14 @@
+{{- define "dogstatsd-sidecar" -}}
+{{- if .Values.dogstatsd.enabled -}}
```

Comment:
Is possible to add this env var:
`DD_HOSTNAME` and set default value to `dogstatsd` ?

Reply:
done